docs/source/api/v2/servers_hostname_update.rst

```diff
@@ -0,0 +1,81 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-servers-id-queue_update:
+
+*************************************
+``servers/{{HostName-Or-ID}}/update``
+*************************************
+
+``POST``
+========
+:term:`Queue` or dequeue updates and revalidation updates for a specific server.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object
+
+Request Structure
+-----------------
+.. table:: Request Path Parameters
+
+	+------------------+---------------------------------------------------------------------------------------------------------+
+	| Name             | Description                                                                                             |
+	+==================+=========================================================================================================+
+	|  HostName-OR-ID  | The hostName or integral, unique identifier of the server on which updates are being queued or dequeued |
+	+------------------+---------------------------------------------------------------------------------------------------------+
+
+.. table:: Request Query Parameters
+
+	+---------------+----------+--------------------------------------------------------------------------------------+
+	| Name          | Required | Description                                                                          |
+	+===============+==========+======================================================================================+
+	| updated       | no       | The value to set for the queue update flag on this server. May be 'true' or 'false'. |
+	+---------------+----------+--------------------------------------------------------------------------------------+
+	| reval_updated | no       | The value to set for the queue update flag on this server. May be 'true' or 'false'. |
+	+---------------+----------+--------------------------------------------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/2.0/servers/my-edge/update?updated=true&reval_updated=false HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+A human-readable string explaining the value(s) updated.
```

Comment:
You can just have the example since there's no `response` object. So basically you can get rid of this line.

Reply:
Done